vale/CockroachDB/NondescriptLinkTitles.yml

```diff
@@ -2,18 +2,30 @@ extends: existence
 message: Avoid using non-descriptive link titles.
 link: https://github.com/cockroachdb/docs/blob/main/StyleGuide.md#lists
 level: warning
-scope: link
+scope: raw
 nonword: true
 tokens:
-  - '^(?i)click here$'
-  - '^(?i)click this$'
-  - '^(?i)find out more$'
-  - '^(?i)go$'
-  - '^(?i)here$'
-  - '^(?i)learn more$'
-  - '^(?i)more$'
-  - '^(?i)over here$'
-  - '^(?i)read here$'
-  - '^(?i)read more$'
-  - '^(?i)right here$'
-  - '^(?i)this$'
+  - '(?i)\[click here\]'
+  - '(?i)\[click this\]'
+  - '(?i)\[find out more\]'
+  - '(?i)\[go\]'
+  - '(?i)\[here\]'
+  - '(?i)\[learn more\]'
+  - '(?i)\[more\]'
+  - '(?i)\[over here\]'
+  - '(?i)\[read here\]'
+  - '(?i)\[read more\]'
+  - '(?i)\[right here\]'
+  - '(?i)\[this\]'
```

Comment:
Maybe also "this link", "that link", "the documentation"?

Reply:
Added!